PublicationsRouter/api/v3/Send.md

```diff
@@ -138,12 +143,12 @@ To do this, send the bare-minimum JSON notification, with only the format identi
 
     POST /validate?api_key=<api_key>
     Header:
-        Content-Type: multipart/form-data; boundary=FulltextBoundary
+        Content-Type: multipart/related; boundary=FulltextBoundary
```

Comment:
Change line 142.  Add a new sentence BEFORE "For example:" (which should then be a new para - i.e. not tagged onto the end of this para).

`This request must be multi-part, formed using [RFC 2387](https://www.ietf.org/rfc/rfc2387.txt).`